src/components/Typography/index.js

```diff
@@ -64,3 +64,108 @@ export const Interaction = {
   )
 }
 
+
+const eSubheadRule = css(styles.editorialSubhead)
+const eLeadRule = css(styles.editorialLead)
+const eCreditRule = css(styles.editorialCredit)
+const ePRule = css(styles.editorialP)
+const eQuestionRule = css(styles.editorialQuestion)
+const eQuoteTextRule = css(styles.editorialQuoteText)
+const eCiteRule = css(styles.editorialCite)
+const eBoxTitleRule = css(styles.editorialBoxTitle)
+const eBoxTextRule = css(styles.editorialBoxText)
+const eAuthorLinkRule = css(styles.editorialAuthorLink)
+const eMark = css(styles.editorialMark)
+const eCaption = css(styles.editorialCaption)
+const eByline = css(styles.editorialByline)
+
+const getEditorialHeadlineRule = (type) => {
+  switch(type) {
+    case 'meta': return css(styles.editorialHeadlineMeta)
```

Comment:
`editorialHeadlineMeta` doesn't make sense—lets make a Interaction.Headline with this style.

Reply:
Done.